packages/dapp/src/pages/index.tsx

```diff
@@ -24,6 +43,60 @@ const Home = () => {
     }
   };
 
+  const init = async () => {
+    if (library && library.getSigner() && contracts) {
+      try {
+        const signer = library.getSigner();
+        const tokenAddress = await contracts.yourReadContract.token();
+        console.log(`tokenAddress`, tokenAddress);
+
+        const tokenContract = new ethers.Contract(
```

Comment:
Maybe we can add it on Web3Provider after initialize the DRecruitV1 contract and change yourReadContract to yourReadContracts (the same for write), and be able to use the token contract from the context.

Reply:
This `contracts` variable from the `Web3Context` consists of DRecruit contract. So I'll add new variable maybe `tokenContract` to `Web3Context` for our token contract